app/enterprise/2.1.x/developer-portal/configuration/authentication/sessions.md

```diff
@@ -60,6 +60,11 @@ The Session configuration is secure by default, so the cookie uses the [Secure,
 * If using HTTP instead of HTTPS: `"cookie_secure": false`
 * If using different subdomains for the [portal_api_url](/enterprise/{{page.kong_version}}/developer-portal/networking/#portal_api_url) and [portal_gui_host](/enterprise/{{page.kong_version}}/developer-portal/networking/#portal_gui_host), see the example below for [Domains](https://docs.konghq.com/enterprise/2.1.x/developer-portal/configuration/authentication/sessions/#domains). 
 
+⚠️**Important:** Sessions are not invalidated when a user logs out if `"storage": "cookie"`
```

Comment:
Suggest using a dev format with coloring to make this stand out more.

  <div class="alert alert-warning">
    <i class="fas fa-exclamation-triangle" style="color:orange; margin-right:3px"></i>
    <strong>Important: </strong> Sessions are not invalidated when a user logs out if `"storage": "cookie"`
    (the default) is used. In that case, the cookie is deleted client-side. Only when session data is 
    stored server-side with `"storage": "kong"` set is the session actively invalidated.
    </div>